packages/miniserve/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/svenstaro/miniserve
+TERMUX_PKG_DESCRIPTION="Tool to serve files via HTTP"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.14.0
+TERMUX_PKG_SRCURL=https://github.com/svenstaro/miniserve/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=68e21c35a4577251f656f3d1ccac2de23abd68432810b11556bcc8976bb19fc5
+TERMUX_PKG_DEPENDS=libbz2
+TERMUX_PKG_BUILD_IN_SRC=true
+termux_step_pre_configure() {
+	rm -f Makefile
+	termux_setup_rust
+}
+termux_step_make() {
+	cargo build --jobs $TERMUX_MAKE_PROCESSES --target $CARGO_TARGET_NAME \
+		--release --locked
+	cargo test --release --locked -- --test-threads=1
```

Comment:
You are cross-compiling from Ubuntu x86_64 to Termux. You will not be able to run tests.